util/Migrator/DbScripts/2022-09-26_00_EventsSystemUser.sql

```diff
@@ -0,0 +1,92 @@
+-- Add column SystemUser to Event table
+IF COL_LENGTH('[dbo].[Event]', 'SystemUser') IS NULL
+    BEGIN
+        ALTER TABLE
+            [dbo].[Event]
+        ADD
+            [SystemUser] TINYINT NULL;
+    END
+GO
+    
+-- Recreate EventView so that it includes the SystemUser column
```

Comment:
Nice catch! I was on the lookout for this 😁

Reply:
It was working when I tested locally and it took me a while to figure this out 😅